src/app/visual/popover/popover-visual.component.html

```diff
@@ -310,12 +292,12 @@
       [skyPopover]="myPopover5"
       skyPopoverPlacement="above"
       skyPopoverAlignment="right">
-      Open popover
+      Above, right
     </button>
   </p>
 </div>
 <div id="screenshot-popover-fullscreen">
-    <button
+  <button
     id="btn-popover-fullscreen"
     class="sky-btn"
     [skyPopover]="popoverFullscreen"
```

Comment:
When I click on the "Show fullscreen popover" button, and then hit `tab`, it closes the popover. Shouldn't it set focus inside of the popover so I can iterate over the tabbable elements?

Reply:
Ugh, you're right. This feature is going to die in the next version of SKY UX (and our design guidelines say to NOT use fullscreen popovers). Let me see if I can come up with something simple to address this.

Reply:
Nice fix.